Server/ChessArena.proto

```diff
@@ -0,0 +1,68 @@
+syntax = "proto3";
+
+service ChessArena {
+    // A simple RPC
+    //
+    // Inform server that the client is ready to start the game. 
+    // Returns one of the following:
+    // 1. Player is starting player (white). Opponent move message not set.
+    // 2. Player is not starting player (black). Opponent move message provides opening
+    //    move coordinates. 
+    rpc Initialize(PlayerInformation) returns (GameStartInformation) {}
+
+    // A Bidirectional streaming RPC.
+    //
+    // Send player moves. Receive opponent moves.
+    rpc CreateMovements(stream PlayerMove) returns (stream OpponentMove) {}
+}
+
+// Initialize
+
+message PlayerInformation {
+    string name = 1;    
+}
+
+message GameStartInformation {
+    // True if client is starting player.
+    // If false, opponent move is also returned.
+    bool whitePlayer = 1;
+
```

Comment:
Maybe these empty lines could be omitted?

Reply:
Same for other messages

Reply:
This is pretty much personal preference to semantics question. I'm used to have space between properties.

Reply:
Empty lines omitted in latest commit.